src/components/Text.js

```diff
@@ -32,6 +39,7 @@ const Text = ({
         color: ${alternate ? COLOUR_SECONDARY_TEXT : COLOUR_PRIMARY_TEXT};
         padding: 2px 1rem;
         line-height: 1.5;
+        font-size: ${fontSizes[size] || 'inherit'};
```

Comment:
Should we consider having a central source of truth? The heading component also controls font size, perhaps the font sizes, small, medium and large, should be configured in theme. So that heading and text have the same values set.